src/cli.cc

```diff
@@ -96,6 +109,30 @@ ExitCode CompleteInstall(AkliteClient& client) {
       exit_code = ExitCode::InstallNeedsRebootForBootFw;
       break;
     }
+    case InstallResult::Status::Failed: {
+      // do rollback
+      // check rollback type, bootloader (boot on the previous version) or App driven
```

Comment:
Not sure if my question makes sense:
- should we ask a user if they want to retry the installation or rollback in case of failure? Maybe, an argument for that?

Reply:
It does. I don't know. I would ask for feedback at first.